docs/manual/core-concepts/raw-queries.md

```diff
@@ -57,27 +57,6 @@ await sequelize.query('SELECT 1', {
 console.log(await sequelize.query('SELECT * FROM projects', { raw: true }));
 ```
 
-## "Dotted" attributes
-
-If an attribute name of the table contains dots, the resulting objects will be nested. This is due to the usage of [dottie.js](https://github.com/mickhansen/dottie.js/) under the hood. See below:
-
-```js
-const rows = await sequelize.query('select 1 as `foo.bar.baz`');
```

Comment:
It works with nest option, please add `nest: true` to this query

Reply:
Indeed, fixed